vllm/worker/model_runner.py

```diff
@@ -138,11 +149,16 @@ def __init__(
 
         # Lazy initialization
         self.model: nn.Module  # Set after load_model
-        # Set if the backend is flashinfer.
-        self.flashinfer_workspace_buffer: torch.Tensor
         # Set after load_model.
         self.lora_manager: Optional[LRUCacheWorkerLoRAManager] = None
 
+        # Flashinfer fields
+        self.flashinfer_prefill_wrapper: Optional[
+            BatchPrefillWithPagedKVCacheWrapper] = None
+        self.flashinfer_decode_wrapper: Optional[
+            Union[BatchDecodeWithPagedKVCacheWrapper,
+                  CUDAGraphBatchDecodeWithPagedKVCacheWrapper]] = None
```

Comment:
I think it would be a good opportunity to see if we can push this down to the attention backend class. we shouldn't have any conditionals depending on the attention type in model runner.